charts/podinfo/templates/tests/jwt.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     sidecar.istio.io/inject: "false"
     linkerd.io/inject: disabled
     appmesh.k8s.aws/sidecarInjectorWebhook: disabled
+    # TODO is disabling osm sidecar injection necessary for this scenario?
```

Comment:
Unless OSM has support for jobs, which I really doubt, then yes we need to disable injection as Envoy makes the job fail.

Reply:
Resolved in c328a3cb919d82609157c3a04369adb30a71050b

Reply:
Added it in the most recent revision. See 

![image](https://user-images.githubusercontent.com/13926417/128921557-e68d6d34-044f-4524-9402-1bb509c122cd.png)


https://github.com/fluxcd/flagger/pull/955/files#diff-d7c1634246fda9a353544e09059fbe39ca04369ed4f9ecc2fd1e410b66807b50R15